v63003/u_inref/gtm8735.csh

```diff
@@ -60,15 +60,17 @@ $DSE dump -file|&$grep "Access method"
 $DSE dump -file|&$grep "Read Only"
 
 
-echo '# Displaying status of gtmhelp databases'
+echo '# Displaying status of gtmhelp databases to verify files have read-only permissions for user/group/other'
 ls -l $ydb_dist/*.dat|awk '{print $1,$9}'
```

Comment:
Need to use $tst_awk (not awk) in test system scripts. Just like $grep (instead of grep)